PCLumiProd/DerivePCCCorrections.py

```diff
@@ -18,7 +18,7 @@
 parser.add_argument("--noType2", action='store_true', default=False, help="Only apply the type1 correction")
 parser.add_argument("-u","--useresponse", action='store_true', default=False, help="Use the final response instead of the real activity to calculate the Type2 Correction")
 parser.add_argument('-b', '--batch',   action='store_true', default=False, help="Batch mode (doesn't make GUI TCanvases)")
-parser.add_argument('-p', '--par', default="0.059,0.0,0.0007,0.013", help="The parameters for type1 and type2 correction (0.059,0.0,0.0007,0.013)")
+parser.add_argument('-p', '--par', default="0.074,0.0,0.00086,0.014", help="The parameters for type1 and type2 correction (0.059,0.0,0.0007,0.013)")
```

Comment:
Can you also change the comment at the end of the line so that it reflects the new default values?